operator/backupcontroller/executor.go

```diff
@@ -69,6 +69,10 @@ func (b *BackupExecutor) listAndFilterPVCs(ctx context.Context, annotation strin
 		return nil, fmt.Errorf("list pods: %w", err)
 	}
 	for _, pod := range pods.Items {
+		if pod.Status.Phase == corev1.PodSucceeded || pod.Status.Phase == corev1.PodFailed {
```

Comment:
Maybe instead of not considdering Succedded and Failed pods, we could switch the logic arround:
only consider pods that are running.

This way we also ignore pods in Pending or Terminating states, which both could also lead to lock ups of jobs.

Reply:
Yes, that's a good point, was focused so far only on the issue right in front of me 😃 

Will do a tweak for this.